spec/controllers/reviews_controller_spec.rb

```diff
@@ -254,49 +254,94 @@ def valid_session
   end
 
   describe "PUT update" do
-    let(:admin_user) { create(:admin_user) }
-    before(:each) do
-      set_current_user admin_user
-    end
-    describe "with valid params" do
-      it "updates the requested review" do
-        review = Review.create! valid_attributes
-        # Assuming there are no other reviews in the database, this
-        # specifies that the Review created on the previous line
-        # receives the :update_attributes message with whatever params are
-        # submitted in the request.
-        Review.any_instance.should_receive(:update_attributes).with({'these' => 'params'})
-        put :update, {:id => review.to_param, :review => {'these' => 'params'}}, valid_session
+    context 'as an admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.